resource/global/YoStarJP/resource/tasks.json

```diff
@@ -259,6 +259,10 @@
                 "ブレイズ",
                 "煌"
             ],
+            [
+                "ブレイス",
+                "煌"
```

Comment:
```suggestion
                "ブレイ[ズス]",
                "煌"
            ],
```
Why not

Reply:
The Japanese version has an exclusive "equivalence_classes" in ocr_config.json, which sometimes oddly doesn't work when the "one of" ([ ]) usage appears in same ocr_replace rule, which I have started to avoid.

I have seen MAA crash due to ocr replace only one time. My impression is that in the tested ocr replacement rules, the replaced text can also be matched by the replacement rules, resulting in repeated ocr replacement and eventual crash.
I noticed that there are some replacement rules in the English and Korean versions that continue to replace non-Chinese content with non-Chinese content, as well as some global rules that replace non-character content into empty. Unexpectedly circular substitutions occur in these rules, may be the truth behind running out of OCR times.
Of course, I will try to reduce unnecessary substitution rules in the future to avoid encountering the real upper limit of substitutions.

Reply:
What was happening with the TraderShopping seemed to have been resolved by reducing it under ~600 ocr replacement cases.
As long we try to reduce what's possible, without going crazy, everything's good.
With EN I'm close to 550 replaces, while JP seems to be almost 700 (and nothing has happened yet luckily).